arrow-libs/fx/arrow-collectors/build.gradle.kts

```diff
@@ -0,0 +1,65 @@
+@file:Suppress("DSL_SCOPE_VIOLATION")
+
+import java.time.Duration
+
+plugins {
+  id(libs.plugins.kotlin.multiplatform.get().pluginId)
+  alias(libs.plugins.arrowGradleConfig.kotlin)
+  alias(libs.plugins.arrowGradleConfig.publish)
+  alias(libs.plugins.spotless)
+  alias(libs.plugins.kotlinx.kover)
+}
+
+apply(from = property("ANIMALSNIFFER_MPP"))
+
+kotlin {
+  sourceSets {
+    commonMain {
+      dependencies {
+        api(projects.arrowFxCoroutines)
+        api(projects.arrowAtomic)
+        api(libs.coroutines.core)
+        implementation(libs.kotlin.stdlibCommon)
+      }
+    }
+
+    commonTest {
+      dependencies {
+        implementation(libs.kotlin.test)
+        implementation(libs.kotest.frameworkEngine)
+        implementation(libs.kotest.assertionsCore)
+        implementation(libs.kotest.property)
+      }
+    }
+  }
+
+  jvm {
+    tasks.jvmJar {
+      manifest {
+        attributes["Automatic-Module-Name"] = "arrow.collectors"
+      }
+    }
+  }
+
+  js {
+    nodejs {
+      testTask {
+        useMocha {
+          timeout = "60s"
+        }
+      }
+    }
+    browser {
+      testTask {
+        useKarma {
+          useChromeHeadless()
+          timeout.set(Duration.ofMinutes(1))
+        }
+      }
+    }
```

Comment:
**Off-topic:** I think would be good if we can put this in some common configuration at some point. Still not sure what's the best way to do that in Gradle 🙃